src/common/inline-menu/InlineMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import {FC} from "react";
+
+type InlineMenuProps = {
+  action: Function;
+};
+
+export const InlineMenu: FC<InlineMenuProps> = ({action}) => {
+  return (
+    <div className="px-24">
+      <ul className="flex justify-between p-2 text-sm font-bold">
+        <li onClick={() => action()} className="flex">
```

Comment:
li isn't an interaction element

add a Button in the li and bind the action `fn`